kong/db/declarative/init.lua

```diff
@@ -241,7 +275,8 @@ function declarative.export_from_db(fd)
   end
 
   fd:write(declarative.to_yaml_string({
-    _format_version = "1.1",
+    _format_version = "2.0",
```

Comment:
does this require an update to docs?

Reply:
Indeed. I have switched the version to 1.2 everywhere.